edge/pkg/devicetwin/dttype/types_helper.go

```diff
@@ -6,6 +6,7 @@ import (
 	"time"
 
 	"github.com/kubeedge/kubeedge/edge/pkg/devicetwin/dtclient"
+	"k8s.io/klog"
```

Comment:
Move to L7

Reply:
It is what gofmt does.

Reply:
"time"

       "k8s.io/klog"

	"github.com/kubeedge/kubeedge/edge/pkg/devicetwin/dtclient"